macos/FluentUI/Button.swift

```diff
@@ -285,20 +286,49 @@ open class Button: NSButton {
 				y = Button.verticalEdgePadding
 				width = rect.width - (Button.horizontalEdgePadding * 2)
 				height = rect.height - (Button.verticalEdgePadding * 2)
-			case .imageLeading:
-				x = isRTL ? titleSize.width + Button.titleAndImageRectInterspacing : 0
+			case .imageLeft:
+				x = Button.horizontalEdgePadding
+				y = Button.verticalEdgePadding
+				width = imageSize.width
+				height = imageSize.height
+				break
+			case .imageRight:
+				x = titleSize.width + Button.titleAndImageRectInterspacing + Button.horizontalEdgePadding
+				y = Button.verticalEdgePadding
+				width = imageSize.width
+				height = imageSize.height
+				break
+			case .imageBelow:
+				x = (rect.width - imageSize.width) / 2
+				y = Button.verticalEdgePadding + titleSize.height + Button.titleAndImageRectInterspacing
+				width = imageSize.width
+				height = imageSize.height
+				break
+			case .imageAbove:
+				x = (rect.width - imageSize.width) / 2
 				y = Button.verticalEdgePadding
-				width = rect.width - Button.titleAndImageRectInterspacing - titleSize.width
+				width = imageSize.width
+				height = imageSize.height
+				break
```

Comment:
We now can handle ALL the enum values.. for whoever ever wants to do this 
![Screen Shot 2020-07-30 at 3 33 16 PM](https://user-images.githubusercontent.com/6722175/88981251-688aee80-d27a-11ea-8db5-37d7395d33b6.png)